pyClient/zeth/utils.py

```diff
@@ -23,11 +23,16 @@
 from typing import List, Tuple, Union, Any, cast
 
 
+WEB3_HTTP_PROVIDER_TIMEOUT = 60 * 60
```

Comment:
Why 60*60?
Let's add the units to the name -> `WEB3_HTTP_PROVIDER_TIMEOUT_SEC`?

Reply:
This was left form debugging.  Renamed and set to 1min which should be much more than enough.